docs/workflows/react-native-specific-workflow-steps/react-native-ui-test.md

```diff
@@ -0,0 +1,121 @@
+---
+title: React Native UI Test
+description: Learn how to run UI tests for your React Native projects easily with Appcircle, ensuring high-quality code and improved app performance.
+tags: [react native, mobile, workflow, step, ui, tests, ui test]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# React Native UI Test
+
+This component runs all the UI tests in your project written with [Detox](https://wix.github.io/Detox/docs/introduction/getting-started) integration. When this step is completed, it generates a test report file in `e2e-report.xml` format. You can view these test results in detail using Appcircle's Test Report component.
+
+For detailed information for Continuous Testing. Please visit our [React Native Continuous Testing documentation](/continuous-testing/react-native-testing/react-native-ui-testing-with-detox).
+
+To generate detailed Test Reports. Please visit our [Test Reports Component documentation](/workflows/react-native-specific-workflow-steps/test-reports-react-native).
+
+:::info Java Version
+
+The default Java version in Appcircle's build stacks is **Java 11**. If your project requires a **higher** Java version, please see the [**How to Change Java Version**](/workflows/common-workflow-steps/custom-script#how-to-change-java-version) document. On the other hand, you can see all details for build stacks both iOS and Android with this [documentation](/infrastructure).
+
+:::
+
+### Prerequisites
+
+The workflow steps that need to be executed before running the **React Native UI Test** workflow step in both iOS and Android platforms, along with their respective reasons, are listed in the table below.
+
+#### For iOS
+
+| Prerequisite Workflow Step                                                                 | Description                                                                                                                                                                                        |
+|--------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Git Clone**](/workflows/common-workflow-steps#git-clone)                                | Clone the selected repository to the build machine. Please use the [**Install Node**](https://docs.appcircle.io/workflows/react-native-specific-workflow-steps#install-node) step after this step. |
```

Comment:
IMO you don't need to specify this because the image is attached and you wrote it in the correct order for Prerequisite.
```suggestion
| [**Git Clone**](/workflows/common-workflow-steps#git-clone)                                | Clone the selected repository to the build machine. |
```